src/Analyzer/FileParser.php

```diff
@@ -3,12 +3,14 @@
 
 namespace Arkitect\Analyzer;
 
+use PhpParser\Lexer\Emulative;
 use PhpParser\NodeTraverser;
 use PhpParser\NodeVisitor\NameResolver;
 use PhpParser\ParserFactory;
 
 class FileParser implements Parser
 {
+    private const PHP_VERSION = '7.1';
```

Comment:
@pfazzi @nelsonkopliku @micheleorselli @fain182 
Here I have hardcoded this PHP version for the parser.
I think that for now is good enough but I would like to open a new issue/pull request if it's a problem.
What do you think?

Reply:
Good enough for me, maybe i would make the name more explicit, it's a PHP version, but why is used the 7.1?

Reply:
Yes, I think that we need to be sure for this version and why .